src/main/java/core/basesyntax/RandomSize.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomSize {
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 46340;
+
+    private Random random = new Random();

```

Comment:
The `Random` instance should be a local variable inside the method `getRandomSize()` if it's not used elsewhere in the class, as per the checklist.